netci.groovy

```diff
@@ -1624,6 +1656,7 @@ def static calculateBuildCommands(def newJob, def scenario, def branch, def isPR
             switch (architecture) {
                 case 'x64':
                 case 'x86':
+                case 'x86lb':
                     if (architecture == 'x86' && os == 'Ubuntu') {
```

Comment:
Doesn't this add legacyjit testing support for Linux? I don't think we want or need that, do we?